src/app/products/fixed-deposit-products/view-fixed-deposit-product/view-fixed-deposit-product.component.html

```diff
@@ -0,0 +1,421 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
```

Comment:
See the components of Loan, Savings, Shares products as examples and update to maintain consistency wherever necessary. 

For example: for some cases where we have min, max, and default values they have been combined, Use of ngIf for conditional fields without creating any extra variable in the component etc.